browser/resources/settings/email_aliases_page/content/email_aliases_list.tsx

```diff
@@ -0,0 +1,181 @@
+  }
+  &:active {
+    background-color: ${color.desktopbrowser.toolbar.button.active};
+  }
+  padding: 0.25em;
+  border-radius: 0.5em;
+`
+
+const MenuButton = styled(Button)`
+  --leo-button-padding: 0;
+  flex-grow: 0;
+  width: 1.5em;
+`
+
+const BorderedCard = styled(Card)`
+  border-top: ${color.material.divider} 1px solid;
+`
+
+const AliasMenuItem = ({ onClick, iconName, text }:
+  { onClick: EventListener, iconName: string, text: string }) =>
+  <leo-menu-item
+    onClick={onClick}>
+    <Row>
+      <Icon name={iconName} />
+      <span>{text}</span>
+    </Row>
+  </leo-menu-item>
+
+const CopyToast = ({ text, tabIndex, children }: { text: string, tabIndex?: number, children: React.ReactNode }) => {
+  const [copied, setCopied] = React.useState<boolean>(false)
+  const copy = () => {
+    navigator.clipboard.writeText(text)
+    setCopied(true)
+    setTimeout(() => setCopied(false), 1000)
+  }
+  return <div
+    tabIndex={tabIndex}
+    onKeyDown={(e) => {
+      if (e.key === 'Enter') {
+        copy()
+      }
+    }}
+    onClick={copy}>
+    <Tooltip text={copied ? getLocale('emailAliasesCopiedToClipboard') : ''} mode="mini" visible={copied}>
+      {children}
+    </Tooltip>
+  </div>
+}
+
+const AliasItem = ({ alias, onEdit, onDelete }: { alias: Alias, onEdit: () => void, onDelete: () => void }) =>
+  <AliasItemRow>
+    <Col>
+      <CopyToast text={alias.email}>
+        <EmailContainer title={getLocale('emailAliasesClickToCopyAlias')}>
+            {alias.email}
+          </EmailContainer>
+        </CopyToast>
+        {(alias.note || alias.domains) &&
+          <AliasAnnotation>
+            {alias.note}
+            {alias.domains && alias.note && <span>. </span>}
+            {alias.domains && formatMessage(getLocale('emailAliasesUsedBy'),
+              { placeholders: { $1: alias.domains?.join(", ") } })}
+          </AliasAnnotation>}
+        </Col>
+      <AliasControls>
+        <CopyToast text={alias.email} tabIndex={0}>
+          <CopyButtonWrapper
+            title={getLocale('emailAliasesClickToCopyAlias')}>
+            <Icon name="copy"/>
+          </CopyButtonWrapper>
+        </CopyToast>
+        <ButtonMenu>
+          <MenuButton slot='anchor-content' kind='plain-faint' size="large">
+            <Icon name="more-vertical" />
+          </MenuButton>
+          <AliasMenuItem
+            iconName="edit-pencil"
+            text={getLocale('emailAliasesEdit')}
+            onClick={onEdit} />
+          <AliasMenuItem
+            iconName="trash"
+            text={getLocale('emailAliasesDelete')}
+            onClick={onDelete} />
+        </ButtonMenu>
+      </AliasControls>
+    </AliasItemRow>
+
+export const AliasList = ({ aliases, onViewChange, onListChange, mappingService }: {
+    mappingService: MappingService,
+    aliases: Alias[],
+    onViewChange: (viewState: ViewState) => void,
+    onListChange: () => void
+  }) =>
+  <BorderedCard>
```

Comment:
hey, in the design this doesn't have a separator, its two separate cards:

https://www.figma.com/design/N0Xsctr5jt7I3Ots0G355V/Email-aliases?node-id=48-7582&m=dev